.github/workflows/release-icu4c-ubuntu.yml

```diff
@@ -60,14 +60,14 @@ jobs:
         source /etc/os-release
         # Get the ICU version in artifact_version
         source icu4j/releases_tools/shared.sh
-        # Convert 76.1 to 76_1
+        # Convert 77.1 to 77_1
         underscore_version=$(echo $artifact_version | sed 's/\./_/g')
         pushd ${RELEASE_FOLDER}
         tar -czf icu4c-${underscore_version}-Ubuntu${VERSION_ID}-x64.tgz icu
         rm -fr icu
         popd
-        mv icu4c/source/dist/icu4c-76_1-d*    ${RELEASE_FOLDER}
-        mv icu4c/source/dist/icu4c-76_1-src.* ${RELEASE_FOLDER}
+        mv icu4c/source/dist/icu4c-77_1-d*    ${RELEASE_FOLDER}
+        mv icu4c/source/dist/icu4c-77_1-src.* ${RELEASE_FOLDER}
```

Comment:
could we wild-card these?

Reply:
Not really.
There are other files in that folder source/dist folder that we don't want to publish.

I'll fix it, but not now.
I want to get the release out.

Reply:
This is the content of the dist folder:
```
icu4c-77_1-data-bin-l.zip
icu4c-77_1-data.zip
icu4c-77_1-docs.zip
icu4c-77_1-release-77-rc-data.zip
icu4c-77_1-release-77-rc-docs.zip
icu4c-77_1-release-77-rc-src.tgz
icu4c-77_1-release-77-rc-src.zip
icu4c-77_1-src.tgz
icu4c-77_1-src.zip
icu4c-data.zip@
icu4c-docs.zip@
icu4c-src.tgz@
icu4c-src.zip@
tmp/
icu4c-77_1-data-bin-b.zip
```


The "files" ending in `@` are symbolic links. Which we don't publish.
So I can't just `mv icu4c-*`
And we don't want the `icu4c-77_1-src.*` files either (because GitHub does that automatically).

Reply:
I have it on my with-list to unify the naming.
Between C, Java, and the rust data export tool we use: `76_1rc`, `76.1rc`, `76-rc`